blit.lisp

```diff
@@ -215,25 +215,39 @@ This variable is sometimes used to disable recording for editing purposes.")
 ;;;    **
 (defmethod blit (canvas (box row-box) x y)
   (let ((padding (row-box-padding box))
-        #+#:ignore
         (height (height box))
-        #+#:ignore
-        (baseline (baseline box))
+	(align (row-box-align box))
```

Comment:
please disable tabs in Emacs, and then do `C-x h M-x untablify`. The indentation is wrong.

Reply:
Indentation still looks wrong. Reindent.